libpod/oci_linux.go

```diff
@@ -417,8 +427,16 @@ func (r *OCIRuntime) createOCIContainer(ctr *Container, cgroupParent string, res
 		}
 		logrus.Debugf("Received container pid: %d", ss.si.Pid)
 		if ss.si.Pid == -1 {
+			data, err := ioutil.ReadFile(ociLog)
+			if err == nil {
+				var ociErr ociError
+				if err := json.Unmarshal(data, &ociErr); err == nil {
+					return errors.Errorf("container create failed: %s", strings.Trim(ociErr.Msg, "\n"))
```

Comment:
I would really like a wrapped error here so people can get a reasonable cause with errors.Cause() out of the API. Something like `ErrOCIRuntime` or `ErrRuntimeFailure`? If we don't want to print that in the error message we can discard the cause in Podman.